File: my_test/Java/Dogland.java
class Dogland {
  boolean bopp;
  int iage;
  String stfav;
  String stname;
  
  Dogland(boolean bopp, int iage, String stfav, String stname) {
    this.bopp = bopp;
    this.iage = iage;
    this.stfav = stfav;
    this.stname = stname;
  }
  
  void outFunction() {
    if (bopp) {
      iage += 11;
    } else {
      iage -= 11;
    }
    if (stfav.length() > 3) {
      if (stname.length() > 3) {
        iage -= 11;
      } else {
        iage += 11;
      }
    }
    else {
      if (stname.length() > 3) {
        iage += 11;
      } else {
        iage -= 121;
      } 
    }
    for (int i = 0; i < 4; i++) {
      System.out.println(iage + " " + stfav + " " + stname + " ");
      iage += iage;
      System.out.println(iage + " " + stname + " " + stfav + " ");
    }  
  }
}

class lastOut {
  public static void main(String[] args) {
    Dogland groomi = new Dogland(false, 3, "brother", "goo");
    groomi.outFunction();
  }
}

File: my_test/python/Dogland.py
class Dogland :
    #생성자
    def __init__(self, bopp, iage, stfav, stname) :
        self.bopp = bopp
        self.iage = iage
        self.stfav = stfav
        self.stname = stname
    def outFunction(self) :
        if self.bopp :
            self.iage += 11
        else :
            self.iage -= 11
        if len(self.stfav) > 3 :
            if len(self.stname) > 3 :
                self.iage -= 11
            else :
                self.iage += 11
        else :
            if len(self.stname) > 3 :
                self.iage += 11
            else :
                self.iage -= 121
        for i in range (0, 4) :
            print(f"{self.iage} {self.stfav} {self.stname} ")
            self.iage += self.iage
            print(f"{self.iage} {self.stname} {self.stfav} ")

groomi = Dogland(False, 3, "brother", "goo")
groomi.outFunction()
